src/shared/settingsholder.cpp

```diff
@@ -164,10 +157,14 @@ QString SettingsHolder::getReport() const {
   QTextStream out(&buff);
   auto settingsKeys = m_settings.childKeys();
   for (const auto& setting : settingsKeys) {
-    if (SENSITIVE_SETTINGS.contains(setting)) {
-      out << setting << " -> <Sensitive>" << Qt::endl;
-      continue;
-    }
+#define SETTING(type, toType, getter, setter, remover, has, key, defvalue, \
```

Comment:
remove the for-stmt, and the `settingsKeys`. You can do everything in the macro. Something like:
` if (has() && !sensitive) {...`